### PR TITLE
vmware-horizon-client: init at 4.10.0-11053294

### PR DESCRIPTION
**Pull request by buckley310**
First package, I hope I got everything right :)

###### Motivation for this change
Add a package for VMware's remote desktop client
https://www.vmware.com/go/viewclients

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/remote/vmware-horizon-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+let
+    version = "4.10.0-11053294";
+
+    vmwareBundle64 = fetchurl {
+        url = https://download3.vmware.com/software/view/viewclients/CART19FQ4/VMware-Horizon-Client-4.10.0-11053294.x64.bundle;
+        sha256 = "21617d8223aad7d282967b07dbf09ec5d5dd389d706063c0b791b20bdfe425bb";
+    };
+
+    vmwareBundleUnpacker = fetchurl {
+        url = https://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/eclass/vmware-bundle.eclass;
```
Comment (symphorien) [inline pkgs/applications/networking/remote/vmware-horizon-client/default.nix]: Is this url guaranteed to be stable when gentoo updates this file ?

**Comment by buckley310** [comment]
I have updated the URL which should lock the unpacking script to its current version if they ever did change it.

File: pkgs/applications/networking/remote/vmware-horizon-client/default.nix (hunk 1 above)
Comment (buckley310) [reply pkgs/applications/networking/remote/vmware-horizon-client/default.nix]: The URL has been updated to be stable.

**Comment by fabianhauser** [comment]
I'd still be interested in this 🙂
What is the reason this PR was closed @buckley310?

**Comment by buckley310** [comment]
@fabianhauser Basically I ended up suddenly needing smartcard support, which I could not get working. So I closed this, as i would not be using it, and switched to a dockerized version of the client.

However, the 8.0 release was huge. It is now simpler to package and all the features I need are working.
I actually wrote this less than a week ago and have been using it with no issues:
https://gist.github.com/buckley310/1ae362687cf4712209a094063cf726b2

I have not yet bothered to do a PR for it.

**Comment by fabianhauser** [comment]
Would be amazing if you could open a PR :)

**Comment by buckley310** [comment]
@fabianhauser  @das-g  https://github.com/NixOS/nixpkgs/pull/96849

**Pull request closed without merging**